examples/demo.jl

```diff
@@ -120,31 +121,29 @@ hyperparameters["model"] = "$(typeof(model).name.wrapper)"
 hyperparameters["trainable parameters"] = sum(length, Flux.params(model));
 
 if nlabels == 1
-    loss(x, y) = Flux.logitbinarycrossentropy(model(x), y)
+    loss(x, y) = Flux.logitbinarycrossentropy(x, y)
     accuracy(ŷ, y) = mean((Flux.sigmoid.(ŷ) .> 0.5) .== y)
 else
-    loss(x, y) = Flux.logitcrossentropy(model(x), y)
+    loss(x, y) = Flux.logitcrossentropy(x, y)
     accuracy(ŷ, y) = mean(Flux.onecold(ŷ) .== Flux.onecold(y))
 end
 
-loss(x::Tuple) = loss(x[1], x[2])
-
-## Training 
-opt = ADAM()
+## Training (we use Optimisers instead of Flux due to bug in Flux below v0.13.11)
+opt_state = Optimisers.setup(Optimisers.Adam(), model)
```

Comment:
The workaround is good but small question: why not target 0.13.11 instead?

My original idea was for 0.13.9 because that would allow the new syntax while having the broadest compatibility. Version 0.13.11 is only 2 patch versions ahead, so it won't make a big difference.

Version 0.14 is a different story. It has several breaking changes and requires the new Julia 1.9. A few of these are with package management - I see you encountered those errors in another commit. So I am hesitant to upgrade to it just yet.

Reply:
Just followed your request to be compatible with 0.13.9 as I thought there is some reason for this specific version.